tests/nnstreamer_decoder_python3/runTest.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+##
+## SPDX-License-Identifier: LGPL-2.1-only
+##
+## @file runTest.sh
+## @author Gichan Jang <gichan2.jang@samsung.com>
+## @date May 06 2021
+## @brief SSAT Test Cases for NNStreamer
+##
+
+if [[ "$SSATAPILOADED" != "1" ]]; then
+    SILENT=0
+    INDEPENDENT=1
+    search="ssat-api.sh"
+    source $search
+    printf "${Blue}Independent Mode${NC}
+"
+fi
+
+# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
+testInit $1
+
+if [ "$SKIPGEN" == "YES" ]; then
+    echo "Test Case Generation Skipped"
+    sopath=$2
+else
+    echo "Test Case Generation Started"
+    python3 ../nnstreamer_converter/generateGoldenTestResult.py 9
+    python3 ../nnstreamer_merge/generateTest.py
+    sopath=$1
+fi
+convertBMP2PNG
+
+flat_version=$(dpkg-query --showformat='${Version}' --show libflatbuffers)
+if [ $flat_version ]; then
+  major=`echo $flat_version | cut -d. -f1`
+  minor=`echo $flat_version | cut -d. -f2`
+  echo "dpkg major: $major minor: $minor"
+else
+  flat_version=$(rpm -q flatbuffers)
+  parsed_version=`echo $flat_version | cut -d- -f2`
+  major=`echo $parsed_version | cut -d. -f1`
+  minor=`echo $parsed_version | cut -d. -f2`
+  echo "rpm major: $major minor: $minor"
+fi
```

Comment:
IMHO, this flatbuffer version checking mechanism looks too platform-dependent.
1. How about using the special files such as ```/etc/tizen-release``` and ```/etc/lsb-release``` to know what the platform is.
2. It would be better to skip the test if the platform is unknown.

Reply:
Thanks. I changed to check flexbuffers module.
```
if python3 -c "from flatbuffers import flexbuffers" &> /dev/null; then
    echo 'flexbuffers module exists.'
else
    echo 'flexbuffers module does not exist'
    report
    exit
fi
```